### DISCUSSION TITLE
using DeepTrack in conda environment

### DISCUSSION THREAD

**Discussion by montarou** [DeepTrackAI/DeepTrack2#216] [General]
Hello DeepTrack Expert
Sorry if my question is trivial... i would like to use miniconda and spyder framework for using deeptrack ? doest that possible ?
I try to found deeptrack in conda  current channel without success. In fact i would like to do a DeepTrack environment in miniconda.
Did some of you any advices ?

**Comment by giovannivolpe** [comment]
Hi,
DeepTrack is currently now available in Conda.
We'll be looking into that in the future, but we don't have a timeline for this yet.
